generators/utils-github-templates/templates/plain/.github/PULL_REQUEST_TEMPLATE/release.md

```diff
@@ -0,0 +1,65 @@
+## Release notes
+
+*List of addressed issues and PRs since the last release*
+
+
+## Checklists
+
+### Release preperation
+
+* [x] Create new `release-x.x.x` branch (based on `develop` branch)
```

Comment:
How to know the new version number in advance?

Reply:
I don't think there is a way to get the next number automatically - at least I don't see an easy way